lib/onelogin/ruby-saml/utils.rb

```diff
@@ -22,7 +22,7 @@ class Utils
       #
       def self.format_cert(cert)
         # don't try to format an encoded certificate or if is empty or nil
-        return cert if cert.nil? || cert.empty? || cert.match(/\x0d/)
+        return cert if cert.nil? || cert.empty? || !cert.ascii_only?
```

Comment:
Maybe I've misinterpreted the comment above and this function isn't meant to support .der in which case.. maybe we can simply remove `|| cert.match(/\x0d/)`?